pkg/storage/stores/shipper/compactor/deletion/mode.go

```diff
@@ -0,0 +1,50 @@
+package deletion
+
+import (
+	"errors"
+)
+
+type Mode int16
+
+var (
+	errUnknownMode = errors.New("unknown deletion mode")
+)
+
+const (
+	Disabled Mode = iota
+	V1            // The existing experimental log deletion that removes whole streams.
```

Comment:
I think v1 is not really very descriptive. I would call it e.g. `deprecated-delete-whole-streams`

Reply:
It's renamed/